garak/probes/base.py

```diff
@@ -274,13 +274,19 @@ def probe(self, generator) -> Iterable[garak.attempt.Attempt]:
         attempts_todo: Iterable[garak.attempt.Attempt] = []
         prompts = list(self.prompts)
         lang = self.lang
-        prompts = self.langprovider.get_text(prompts)
+        # account for visual jailbreak until Turn/Conversation is supported
+        if isinstance(prompts[0], str):
+            prompts = self.langprovider.get_text(prompts)
+        else:
+            for prompt in prompts:
+                prompt["text"] = self.langprovider.get_text(prompt["text"])
         lang = self.langprovider.target_lang
         for seq, prompt in enumerate(prompts):
+            prompt_text = prompt
+            if not isinstance(prompt, str):
+                prompt_text = self.prompts[seq]["text"]
             notes = (
-                {"pre_translation_prompt": self.prompts[seq]}
-                if lang != self.lang
-                else None
+                {"pre_translation_prompt": prompt_text} if lang != self.lang else None
```

Comment:
black churn again. suspect something is not reading instructions in `pyproject.toml` as intended.

Reply:
This was not `black` format churn the length of the `dict` changed allowing all to fit in the line boundary length, the revert of attempting to extract `["text"]` will reset this formatting.